src/main/java/tools/map/xml/creator/MapXmlHelper.java

```diff
@@ -456,438 +463,327 @@ static void clearGameSettings() {
   ///////////////////////////////////////////
   // Start of XML parsing methods
   ///////////////////////////////////////////
-  static GameStep parseValuesFromXml(final Document dom) {
+  static GameStep parseValuesFromXml(final String gameXmlPath)
+      throws FileNotFoundException, SAXException, IOException, ParserConfigurationException {
     initializeAll();
+    Wrapper<GameStep> stepToGo = new Wrapper<>(MapXmlCreator.GAME_STEP_FIRST);
 
-    final Node mainlastChild = dom.getLastChild();
-    if (!mainlastChild.getNodeName().equals(XML_NODE_NAME_GAME)) {
-      throw new IllegalArgumentException(
-          "Last node of XML document is not the expeced 'game' node, but '" + mainlastChild.getNodeName() + "'");
-    }
-    return parseGameNode(mainlastChild);
-  }
+    new GameParser(gameXmlPath).parse(new FileInputStream(gameXmlPath), new TripleaHandler(null) {
```

Comment:
This `TripleaHandler` implementation currently has 0% test coverage.  I'm assuming that's because you haven't replaced calls to `GameParser#parse(InputStream, AtomicReference<String>, boolean)` in the unit tests with `GameParser#parse(InputStream, DefaultHandler)` yet.  It would be interesting to see how much test coverage that will provide.

Since we're introducing a totally new parser, we probably want to ensure high instruction and branch coverage.  `GameParser` currently has 66% instruction and 55% branch coverage, so that should be the minimum we strive for.